dotcom-rendering/src/lib/querystring.ts

```diff
@@ -1,12 +1,4 @@
 const constructQuery = (query: { [key: string]: any }): string =>
-	Object.keys(query)
-		.map((param: string) => {
-			const value = query[param];
-			const queryValue = Array.isArray(value)
-				? value.map((v) => encodeURIComponent(v)).join(',')
-				: encodeURIComponent(value);
-			return `${param}=${queryValue}`;
-		})
-		.join('&');
+	new URLSearchParams(query).toString();
```

Comment:
on a closer look, I'm not sure why this isn't flagged by typescript. I think the type of `query` and the constructor argument don't match

Reply:
Don’t you want to mark `constructQuery` as deprecated, if all it does is returning an `URLSearchParams`?

Reply:
Yea, I'd be tempted to remove this abstraction completely in favour of the standard API. A little more work, but fewer abstractions.

Reply:
I agree, deprecating `constructQuery` and possibly refactoring existing usages to `URLSearchParams` is the right thing to do. This turned into something a bit bigger than I'd anticipated, but I'll see if it's possible to refactor.